db/tasks.php

```diff
@@ -28,10 +28,10 @@
     array(
         'classname' => '\logstore_xapi\task\emit_task',
         'blocking' => 0,
-        'minute' => '*/5',
+        'minute' => '*/1',
```

Comment:
This doesn't make sense to add to settings because it's a default setting that's already customizable in the standard Moodle settings. Also note that this will never run more often than the cron is configured to run so my vote is leave this as running every minute by default. 

Happy to explain how this works in more detail if helpful.